vm_exec.h

```diff
@@ -169,10 +169,20 @@ default:                        \
 #define THROW_EXCEPTION(exc) return (VALUE)(exc)
 #endif
 
+// Run the interpreter on JIT
```

Comment:
I think the comment here should be reworded. Same for the next comment. `Run the interpreter from the JIT` ?